src/chaplin/views/layout.coffee

```diff
@@ -104,6 +104,10 @@ module.exports = class Layout extends View
 
   # Handle all clicks on A elements and try to route them internally.
   openLink: (event) =>
+    # If in Builder editing mode, do not open link
+    searchParams = new URLSearchParams(window.location.search)
+    return if searchParams.get('__builder_editing__') == 'true'
```

Comment:
The Builder team told us that in Preview mode they would cancel event propagation on links. Is that not true?

The reason I ask is that Builder seems to have different behavior on the _embedded_ vs the _standalone_ previews. You can see this by previewing some mobile content like https://builder.io/content/4f34fd99e96043eb9ba5bce025c5cbeb. React-Router doesn't know anything about `__builder_editing__` but it does know to skip routing when the event is canceled.

Reply:
@jamesarosen Since the event here is a jquery event, we've tried `return if event.stopPropagation();` and `return if event.isDefaultPrevented();` but both did not seem to prevent the link from opening when clicked.